docs/can-canjs/can-api.md

```diff
@@ -283,21 +272,9 @@ _Useful libraries that add important features or extend the core collection._
 - **[can-fixture]** <small><%can-fixture.package.version%></small> Simulate AJAX requests
   - `npm install can-fixture --save`
   - <a class="github-button" href="https://github.com/canjs/can-fixture">Star</a>
-- **[can-fixture-socket]** <small><%can-fixture-socket.package.version%></small> Simulate sockets
```

Comment:
I think `can-fixture-socket` should still be compatible with 4.0. Its dependencies are the same as `canjs/canjs#master`.

Reply:
It’s not included in the build or site right now, so I’m going to remove it and see if it can be added back in with https://github.com/canjs/canjs/issues/4142